src/_guides/language/effective-dart/usage.md

```diff
@@ -119,6 +119,60 @@ library. Follow these two rules:
 * An import path should never contain `/lib/`.
 * A library under `lib` should never use `../` to escape the `lib` directory.
 
+## Booleans
+
+### DO use `??` to convert `null` to a Boolean value.
```

Comment:
Boolean value -> boolean value
(https://developers.google.com/style/word-list#letter-B, and it's what we do elsewhere in this doc.)